.github/workflows/ci.yml

```diff
@@ -10,6 +10,10 @@ on:
 jobs:
   test-linux:
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
```

Comment:
Let's keep fail-fast on

Reply:
Done

Reply:
What I meant was, can we remove this line